test/letters/containers/LetterList.unit.spec.jsx

```diff
@@ -46,6 +46,21 @@ describe('<LetterList>', () => {
     });
   });
 
+  it('renders DL buttons for all letters in list', () => {
```

Comment:
Eventually, once the logic is fixed up above, I think we'll want another test here that makes sure if the letter is BSL and options aren't available that the download button doesn't render.

Reply:
Agreed, will write one up